[2024-02-01 17:55:12,081] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: project_6_24.dim_mysql manual__2024-02-01T17:54:59.362619+00:00 [queued]>
[2024-02-01 17:55:12,110] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: project_6_24.dim_mysql manual__2024-02-01T17:54:59.362619+00:00 [queued]>
[2024-02-01 17:55:12,110] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-02-01 17:55:12,111] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-02-01 17:55:12,111] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-02-01 17:55:12,170] {taskinstance.py:1377} INFO - Executing <Task(MySqlOperator): dim_mysql> on 2024-02-01 17:54:59.362619+00:00
[2024-02-01 17:55:12,189] {standard_task_runner.py:52} INFO - Started process 10360 to run task
[2024-02-01 17:55:12,196] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'project_6_24', 'dim_mysql', 'manual__2024-02-01T17:54:59.362619+00:00', '--job-id', '385', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp_1m6j7id', '--error-file', '/tmp/tmp7brbm802']
[2024-02-01 17:55:12,198] {standard_task_runner.py:80} INFO - Job 385: Subtask dim_mysql
[2024-02-01 17:55:12,560] {task_command.py:369} INFO - Running <TaskInstance: project_6_24.dim_mysql manual__2024-02-01T17:54:59.362619+00:00 [running]> on host 1fafaef2f352
[2024-02-01 17:55:12,964] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=project_6_24
AIRFLOW_CTX_TASK_ID=dim_mysql
AIRFLOW_CTX_EXECUTION_DATE=2024-02-01T17:54:59.362619+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-01T17:54:59.362619+00:00
[2024-02-01 17:55:12,965] {mysql.py:82} INFO - Executing: CREATE TABLE IF NOT EXISTS case_table (
    id INT ,
    status_name VARCHAR(255) NOT NULL,
    status_detail VARCHAR(255) NOT NULL
);


-- CREATE TABLE IF NOT EXISTS province_table (
--     province_id INT PRIMARY KEY,
--     province_name VARCHAR(255) NOT NULL
-- );

-- CREATE TABLE IF NOT EXISTS district_table (
--     district_id INT PRIMARY KEY,
--     province_id INT,
--     district_name VARCHAR(255) NOT NULL,
--     FOREIGN KEY (province_id) REFERENCES province_table(province_id)
-- );


CREATE TABLE IF NOT EXISTS province_table AS
select DISTINCT kode_prov, nama_prov  from raw_covid_data;

CREATE TABLE IF NOT EXISTS district_table AS
select distinct kode_kab, kode_prov , nama_kab from raw_covid_data;
[2024-02-01 17:55:13,050] {base.py:68} INFO - Using connection ID 'mysql_data_staging' for task execution.
[2024-02-01 17:55:13,170] {dbapi.py:208} INFO - Running statement: CREATE TABLE IF NOT EXISTS case_table (
    id INT ,
    status_name VARCHAR(255) NOT NULL,
    status_detail VARCHAR(255) NOT NULL
);


-- CREATE TABLE IF NOT EXISTS province_table (
--     province_id INT PRIMARY KEY,
--     province_name VARCHAR(255) NOT NULL
-- );

-- CREATE TABLE IF NOT EXISTS district_table (
--     district_id INT PRIMARY KEY,
--     province_id INT,
--     district_name VARCHAR(255) NOT NULL,
--     FOREIGN KEY (province_id) REFERENCES province_table(province_id)
-- );


CREATE TABLE IF NOT EXISTS province_table AS
select DISTINCT kode_prov, nama_prov  from raw_covid_data;

CREATE TABLE IF NOT EXISTS district_table AS
select distinct kode_kab, kode_prov , nama_kab from raw_covid_data;, parameters: None
[2024-02-01 17:55:13,183] {dbapi.py:216} INFO - Rows affected: 0
[2024-02-01 17:55:13,303] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=project_6_24, task_id=dim_mysql, execution_date=20240201T175459, start_date=20240201T175512, end_date=20240201T175513
[2024-02-01 17:55:13,385] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-02-01 17:55:13,653] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
